packages/subgraph/src/store-cache.ts

```diff
@@ -1,4 +1,4 @@
-import { BigInt } from "@graphprotocol/graph-ts";
+import { BigInt, Address } from "@graphprotocol/graph-ts";
```

Comment:
I noticed that we don't use this import in this file. Should it be removed instead?